partner_mq_ipf/__manifest__.py

```diff
@@ -21,15 +21,15 @@
 
 {
     'name': "Partner MQ/IPF-update dispatcher",
-    'version': '12.0.0.3.3',
+    'version': '12.0.0.3.4',
     'category': '',
     'description': """
 Listen for updates on the MQ-bus
 v12.0.0.3.0 - Changed sync to af_aisf_jobseeker__sync.
 v12.0.0.3.1 AFC-2467: Implemented queue limit.
 v12.0.0.3.2 AFC-2556: Added different requests to communicate with mq.
-v12.0.0.3.3 AFC-2664: Now skips already synced jobseekers.
-
+v12.0.0.3.3 AFC-2663: Now skips already synced jobseekers.
```

Comment:
Is this issue number correct, same as on the row below.
I believe that it should be AFC-2664?

Reply:
It was a mistake in the manifest file, see branch name and:
https://jira.arbetsformedlingen.se/browse/AFC-2663